CheckDrive.Web/CheckDrive.Web/Views/Cars/Edit.cshtml

```diff
@@ -88,11 +88,11 @@
             </div>
 
             <div class="form-group mt-3">
-                <a asp-action="Details" asp-route-id="@Model?.Id" class="btn btn-outline-info">
+                <a asp-action="Details" asp-route-id="@Model?.Id" class="btn btn-info">
                     <i class="fa-solid fa-arrow-left-long"></i>
                     Orqaga
                 </a>
-                <button class="btn btn-outline-success">
+                <button class="btn btn-success">
```

Comment:
Was there button worked ? it be type="submit"

Reply:
<button> HTML tag's default type is submit

Reply:
OK , i try this other mvc and its does not worked ,